<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>customPlan</class>
 <widget class="QDialog" name="customPlan">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>520</width>
    <height>332</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Custom Plan Generator</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>7</x>
     <y>6</y>
     <width>501</width>
     <height>311</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="6" column="0">
     <widget class="QPushButton" name="generate">
      <property name="text">
       <string>Generate</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QPushButton" name="startTrip">
      <property name="text">
       <string>Start Trip</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLabel" name="label_3">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>11</height>
       </size>
      </property>
      <property name="text">
       <string>Current Plan:</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QTextBrowser" name="textBrowser"/>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label_2">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16</height>
       </size>
      </property>
      <property name="text">
       <string>Starting city:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QTextBrowser" name="startingCity">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>23</height>
       </size>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:7.8pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:8.25pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>11</height>
       </size>
      </property>
      <property name="text">
       <string>Select all cities to travel to:</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QListWidget" name="listWidget">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>202</height>
         </size>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::MultiSelection</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>11</height>
         </size>
        </property>
        <property name="text">
         <string>OR - Select NUMBER of cities to travel to:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="spinBox"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
